dashboard-new/src/app/getWhoami.ts

```diff
@@ -0,0 +1,47 @@
+import lhConfig from '@/lhConfig'
+import { ACLAction, ACLResource, Principal, ServerACLs } from 'littlehorse-client/dist/proto/acls'
+import { getServerSession } from 'next-auth'
+import { WhoAmI } from '../types'
+import { authOptions } from './api/auth/[...nextauth]/authOptions'
+
+const getWhoAmI = async (): Promise<WhoAmI> => {
+  const session = await getServerSession(authOptions)
+  if (session) {
+    const { accessToken } = session
+    const client = lhConfig.getClient(accessToken)
+
+    const { perTenantAcls, globalAcls } = await client.whoami({})
+
+    const tenants = getTenants({ perTenantAcls, globalAcls })
+
+    return {
+      user: session.user,
+      tenants,
+    }
+  } else {
+    return {
+      user: {
+        name: 'anonymous',
+        email: 'anonymous',
+      },
+      tenants: ['default'],
+    }
+  }
+}
+
+const getTenants = ({ perTenantAcls, globalAcls }: Pick<Principal, 'globalAcls' | 'perTenantAcls'>): string[] => {
+  let tenants: string[] = []
+  if (globalAcls && hasDefaultAccess(globalAcls)) {
+    tenants = ['default']
```

Comment:
For this, we need to add an `rpc SearchTenants` and the Principal should have access to all tenants. CC @eduwercamacaro